### PR TITLE
Fix: Prevent scrolling of container (fixes #254)

### LINKED ISSUE #254: Scrolls occasionally
### Subject of the issue
Sometimes when using a screen reader or keyboard controls, as the focus lands on the right button, depending on its styling, the `narrative__slide-container` will scroll into its overflow.

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes #254

### Fix
* Force prevent scroll of image container

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Comment by kirsty-hames** [comment]
I'm unable to replicate this issue testing with the JSX PR https://github.com/adaptlearning/adapt-contrib-hotgraphic/pull/256 Could you confirm if this is still an issue please @oliverfoster ?

File: js/NarrativeView.js [hunk 1]
```diff
@@ -314,6 +320,10 @@ class NarrativeView extends ComponentView {
       this.setupInviewCompletion('.component__widget');
     }
   }
+
+  preRemove() {
+    this.$('.narrative__slide-container')[0].removeEventListener('scroll', this.onScroll, true);
```
[the remark this answers is not in the document]
Comment (oliverfoster) [reply js/NarrativeView.js:325]: Nope, but will `?.` it anyway.

**Review by eleanor-heath — APPROVED** [review]

**Comment by oliverfoster** [comment]
Implicit +1 from conversations with @kirsty-hames

**Pull request merged**